woocommerce/woocommerce-ios
# Update Buildkite Xcode image version

Closes: #6877


### Description

With this PR we migrate the Buildkite Xcode image version to 13.3.1 so we can start using the functionality released with that one.
As you can see  [here](https://buildkite.com/automattic/woocommerce-ios/builds/5587), the pipeline succeeded both in creating a build and passing the unit tests when using functionality (`async/await`) available at that point.

### Testing instructions

You can double-check that [this commit](https://github.com/woocommerce/woocommerce-ios/pull/6884/commits/c72cc86996b5e6fa621edb7dc67986427bf6f07b) succeeded in Buildkite. Please feel free to create new commits or Pull Requests for further testing.

### Screenshots



---
